tools/webpack.config.ts

```diff
@@ -81,6 +144,11 @@ export default (env?: string) : webpack.Configuration => {
     if (production) {
         config.plugins = [
             new BundleTracker({filename: 'webpack-stats-production.json'}),
+            // Extract CSS from files
+            new MiniCssExtractPlugin({
+                filename: "[name][hash].css",
```

Comment:
We should use a `[name].[hash].css` naming scheme with a clear delimiter between the filename and the hash, right?

Reply:
We should start using  `[contenthash]`. I think I works better.

Reply:
@priyankp10 can you explain your reasoning?

Reply:
The content hash is the hash generated from the text content it consumed, so it only invalidates the hash if the file is changed, rather than the whole thing.

I am try to explain it the best of it as i can, it hash to do with how webpack's entry point or our dependencies (in this case using `@import`) and hash are generated  -- contenthash does the better job for it.

To summarize, just provides better hashing.

(I look online for something that can explain this, [this article](https://survivejs.com/webpack/optimizing/adding-hashes-to-filenames/) does that pretty well:  )